src/services/nnid/index.ts

```diff
@@ -2,6 +2,7 @@
 
 import express from 'express';
 import subdomain from 'express-subdomain';
+import path from 'path';

```

Comment:
When importing modules the preferred convention is:

- Native Node modules first
- 3rd party modules
- Our modules

Also native Node modules should be imported with the `node:` prefix